Cargo.toml

```diff
@@ -9,30 +9,30 @@ ui = []
 
 [dependencies]
 ape = "0.2.0"
-app_dirs = "1.1.1"
+app_dirs = "1.2"
```

Comment:
As you suggested, I would prefer reverting this to avoid breaking Windows user when they run the upgrade installer for the next version. I don't think the upgrade brings anything meaningful either so I'm happy to keep the old version for now.

Reply:
Done. I still think the new location is better, but there are more improvements that can be done here, so we might as well wait for those.

I tried to make a system-wide Linux install, and it came out as a bit of a mess.